libsolidity/interface/CompilerStack.cpp

```diff
@@ -630,7 +630,7 @@ bool CompilerStack::isRequestedContract(ContractDefinition const& _contract) con
 	return false;
 }
 
-bool CompilerStack::compile(State _stopAfter)
+bool CompilerStack::compile(State _stopAfter, bool performLinking)
```

Comment:
I think you should either never link or use the `_stopAfter` parameter and add a linking stage.

Reply:
performLinking is set to true in CompilerStack.h. It's only there so I can avoid linking in that function, and do it later in CommandLineinterface to get the used libraries.